### PR TITLE
Update noVNC and add viewonly settings

### PR DESCRIPTION
**Pull request by QcFe**
# Description

This PR updates novnc to [v1.3.0](https://github.com/novnc/noVNC/releases/tag/v1.3.0) and adds `readonly` and `noresize` hash flags.

### CODE REVIEW

File: provisioning/containers/gui-common/novnc/Dockerfile [hunk 1]
```diff
@@ -16,18 +16,29 @@ getConfigVar = (a,b) => {\
...
 if (pathname) UI.forceSetting('path', pathname+'/vnc');\
-UI.forceSetting('resize', 'remote');\
 UI.forceSetting('show_dot', true);\
 UI.forceSetting('reconnect', true);\
-UI.forceSetting('reconnect_delay', 100);\
+UI.forceSetting('reconnect_delay', Math.floor(Math.random()*2000));\
+window.onhashchange();\
 }" >> ${HTML_DATA}/app/ui.js
 
+RUN echo #noVNC_container
```
Comment (giorio94) [inline provisioning/containers/gui-common/novnc/Dockerfile:38]: Suppose this line should be removed

**Review by giorio94 — APPROVED** [review]

**Comment by QcFe** [comment]
Superseded by #749

**Pull request closed without merging**